### ISSUE TITLE
Support Rails 7

### ISSUE THREAD

**Issue by CromonMS** [sshaw/ddex#19]
Hi,

Getting this error when trying to update to Rails 7
```
  In Gemfile:
    ddex (~> 0.0.8) was resolved to 0.0.8, which depends on
      activesupport (>= 3.0, < 7)

    rails (~> 7.0.2.3) was resolved to 7.0.2.3, which depends on
      activesupport (= 7.0.2.3)
```

Are there any tasks that need doing before you can bump activesupport version on the gem?

Thanks.

**Comment by sshaw** [comment]
Hello, first thought is maybe roxml needed < 7 but maybe not? Could just be that version 6 was the latest version at the time and I was trying to prevent future breakage. Can you try removing  `< 7` from [this line](https://github.com/sshaw/ddex/blob/ea976d60c5f72e0fea0f5f3239a2125509bd3dbd/ddex.gemspec#L18) then `bundle install && rake` to make sure tests pass? In your `Gemfile` you can then do `gem "ddex", :path => "/path/to/repo/without/active_support<7"` to use in in your Rails application.

Otherwise I can take a look in a week or so but if the above does work a PR would be great. If not I can probably make those changes myself sooner than 1 week but anything in-depth will require more time from me.

**Comment by CromonMS** [comment]
Sure I'll take a look at it when I get a chance, should be in the next few days!

Thanks

**Comment by CromonMS** [comment]
Looks good:

```
Finished in 5 minutes 0 seconds (files took 5.05 seconds to load)
77 examples, 0 failures
```

No probs running an XML generation on rails 7.0 upgrade branch on my app.

Would you like me to bump to 0.0.9? or just pull request that little change as is?

**Comment by sshaw** [comment]
> Would you like me to bump to 0.0.9?

Normally I'd say no but in this case: yes

Thanks!

**Comment by sshaw** [comment]
Fixed by #21.
